### ISSUE TITLE
Trufont download link doesn't work

### ISSUE THREAD

**Issue by RafalBuchner** [trufont/trufont#728]
like in the subject. Trufont download link doesn't work on the [website](https://trufont.github.io). I hit `Mac OS Install` button and I'm getting a blank page with the text "Not found"

**Comment by madig** [comment]
Yeah, I deleted them because they were outdated and didn't feel like updating the actual site yet.

**Comment by CanRau** [comment]
Would be cool to have an up to date macOS installer 😇

**Comment by matthewstrasiotto** [comment]
lol guess not

**Comment by alerque** [comment]
It looks like the site source is in [this repository](https://github.com/trufont/trufont.github.io).
